proposal.md

```diff
@@ -15,6 +15,14 @@ To populate our dashboard, we will be using the [Kaggle Data Scientists Salaries
 - **FormalEducation** - The level of formal education attained (ordinal with 7 levels: (1) Doctoral degree, (2) Master's degree, (3) Bachelor's degree, (4) Professional degree, (5) Some college/university study without earning a bachelor's degree, (6) I did not complete any formal education past high school, and (7) I prefer not to answer).
 
 To see the other features in the dataset, please see.
+
 ### Research Questions and Usage Scenarios
 
-#### Usage Scenario: 
\ No newline at end of file
+Our app will provide answers to two research questions:
+
+- What Data Science related job titles are more popular in each country and how much they pay on average?
```

Comment:
Sorry! I just realised that this bullet point also refers to job titles which we don't have data on. Maybe we can replace this with education level, or one of the other features in our dataset?

I'll also update my section to include a description of the remote working feature since it's mentioned here.